frontend/components/zuzalu-hypercert-treemap.compat.d.ts

```diff
@@ -0,0 +1,17 @@
+// Provided for the zuzalu-hypercert-treemap.
```

Comment:
Where should we put these? I tried to give it a reasonable name. observablehq provides no typing.

Reply:
Feel free to set the pattern :-)
We haven't had to do that yet